data/json/items/comestibles/bread.json

```diff
@@ -86,100 +86,102 @@
     "type": "COMESTIBLE",
     "id": "flatbread",
     "name": "flatbread",
-    "weight": 56,
+    "weight": 85,
     "color": "brown",
     "spoils_in": "11 days 16 hours",
     "container": "bag_plastic",
     "comestible_type": "FOOD",
     "symbol": "%",
-    "calories": 174,
+    "calories": 234,
     "description": "Simple unleavened bread.",
     "price": 140,
     "material": "wheat",
-    "volume": 2,
-    "charges": 6,
```

Comment:
Because you changed the charges of this item, you need to divide the weight and price by the number of charges.

In bread.json, you change charges on lines (other than this):
119, 141, 162, 184, 204, 226, and 244